packages/bootstrap-vue-next/src/components/BTable/BTable.vue

```diff
@@ -9,6 +9,8 @@
     :field-column-class="getFieldColumnClasses"
     @head-clicked="onFieldHeadClick"
     @row-clicked="onRowClick"
+    @row-hovered="(row, index, e) => emit('row-hovered', row, index, e)"
+    @row-unhovered="(row, index, e) => emit('row-unhovered', row, index, e)"
```

Comment:
```suggestion
    @row-hovered="(row, index, e) => { emit('row-hovered', row, index, e) }"
    @row-unhovered="(row, index, e) => { emit('row-unhovered', row, index, e) }"
```
This will have a lint issue that will need to be fixed.